benchmarks/random_variables.py

```diff
@@ -88,18 +87,18 @@ class Sampling:
     Benchmark sampling routines for various distributions.
     """
 
-    param_names = ["dist"]
-    params = [distribution_names]
+    param_names = ["rv_name"]
```

Comment:
Here I'd suggest to use `"rv"` since this actually shows up in the benchmark interface as the identifier.

Reply:
Alternatively "randvar"